handlers.go

```diff
@@ -2958,7 +2961,9 @@ func (s *Server) handleLoginUser(c echo.Context) error {
 		return err
 	}
 
-	if user.PassHash != body.PassHash {
+	
+	
+	if user.PassHash != util.GetPasswordHash(body.Password, user.Salt) {
```

Comment:
@kelindi @alvin-reyes we need a backward compatible check for accounts that are already in plaintext